packages/integration-tests/__tests__/defined-behavior/2-expected-use/custom-operations.ts

```diff
@@ -290,7 +291,26 @@ describe('custom operations', () => {
   >;
   // #endregion
 
-  test.skip('primitive type result', async () => {
+  test('schema.transform() includes custom types, ref types, and operations', () => {
+    const transformedSchema = schema.transform();
+    const expectedTypes = ['CustomArgType', 'EchoResult', 'Query', 'Mutation'];
+    const expectedOperations = [
+      'queryWithCustomTypeArg(customArg: ID): String',
+      'queryWithRefArg(refArg: ID): String',
+      'mutateWithCustomTypeArg(customArg: ID): String',
+      'mutationWithRefArg(refArg: ID): String',
```

Comment:
Similarly to previous comments, these should all be input types, something like:
```ts
'queryWithCustomTypeArg(customArg: QueryWithCustomTypeArgInput): String',
'queryWithRefArg(refArg: QueryWithRefArgInput): String',
'mutateWithCustomTypeArg(customArg: MutateWithCustomTypeArgInput): String',
'mutationWithRefArg(refArg: MutationWithRefArgInput): String'
```

And we should also test that these Input types themselves are being emitted in `schema.transform()`